### DISCUSSION TITLE
Black Screen Wallpaper

### DISCUSSION THREAD

**Discussion by odorizzioficial** [Taiizor/Sucrose#78] [Q&A] [answered]
![image](https://github.com/Taiizor/Sucrose/assets/110546717/c396b8a1-7603-4018-be31-cba9e808c943)

Hello, tried to use the program but when I add the Wallpaper I get a black screen on Windows 10 and Windows 11 and the settings of my computer and notebook are fine and the SDK is also installed

**Comment by odorizzioficial** [comment]
And can I add a video about Sucrose your App to my YouTube channel?

**Reply by Taiizor** [reply]
Of course, you can add it.

When you refer to a black screen, you're talking about the video not playing. There could be several reasons for this. The video could be private or set to not play outside of YouTube. You might rarely encounter this error. When you do encounter it, closing and reopening the wallpaper should fix it.

**Reply by Taiizor** [reply, marked as answer]
The video at this address is a private video, so you'll encounter the error: https://www.youtube.com/watch?v=3m11vEJDdBM
However, this video will work fine when applied as a wallpaper: https://www.youtube.com/watch?v=skjSyV7cLEI

**Reply by odorizzioficial** [reply]
Perfect, I found the problem haha, and is there a way to start the application together with Windows when turning on the computer? I didn't find this option

**Reply by Taiizor** [reply]
Yes, there are, and with many options as well.

![Sucrose-Wallpaper-Engine](https://github.com/Taiizor/Sucrose/assets/41683699/6f8663e2-dc7a-4e2a-9d8e-307b518a4f1b)

**Reply by odorizzioficial** [reply]
Perfect <3 I'd like to make some suggestions for improvement:

1º
Translation: Brazilian Portuguese people here love this software, but most of it is never translated into our language.

2º
When adding a wallpaper, remove the obligation to fill in the fields of: Title, Description, Author etc....if it is for personal use, for sharing yes, you leave it as it is.

3º Make it easier for the user to send a wallpaper for approval to put in the main store for everyone

4 º Improve the optimization of the interface in general, I found it a bit heavy frankly 

5º Put the (i) of information in some functions because it is difficult to understand what it is for or to place the mouse over a function, example: "the scheduler is used to start together with the system".

The rest is perfect

**Reply by Taiizor** [reply]
1. I will soon add machine translation for the Portuguese language.
2. Actually, even if it's for personal use, these pieces of information need to be filled in. Other than that, the optional fields, which should already be optional, don't need to be filled in as they should.
3. Currently, it's available on the GitHub Store, so there isn't really an easier solution. Perhaps if this changes in the future, it will be easier.
4. You're right about the interface. There are optimization issues.
5. I've noted it down, and I'll write detailed explanations for the necessary parts later on.

**Reply by odorizzioficial** [reply]
Perfect, but otherwise, it's perfect, everything is well organized and the layout is beautiful, success to you

**Reply by Taiizor** [reply]
The Portuguese machine translation is ready.

![SucroseWallpaperEngine](https://github.com/Taiizor/Sucrose/assets/41683699/1c6eac59-ae7d-47be-b318-a56c024a0a8a)
![SucroseWallpaperEngine](https://github.com/Taiizor/Sucrose/assets/41683699/8e1e53d0-58c2-43bf-8615-a3b72557602a)
![SucroseWallpaperEngine](https://github.com/Taiizor/Sucrose/assets/41683699/c07e5cdc-efb7-42d6-ae80-61121d4f9141)

**Reply by Taiizor** [reply]
The update including the Portuguese translation has been released.

**Reply by odorizzioficial** [reply]
Thank you very much my friend, I've posted the video on my channel to help you :) https://www.youtube.com/watch?v=tiKYyXCGF4A

**Reply by Taiizor** [reply]
This has been the most detailed introduction I've seen so far. Of course, it would have been nice if the Portuguese translation had been ready for the introduction, but it didn't happen (:
Thank you very much as well.

**Reply by odorizzioficial** [reply]
I'll redo the video in the future, hugs

**Reply by Taiizor** [reply]
I'll be eagerly waiting (;
